### DISCUSSION TITLE
paragraphs detection

### DISCUSSION THREAD

**Discussion by sixofour** [deepgram/community#613] [General help] [answered]
Does anyone know how exactly Deepgram splits the transcripts into paragraphs? Is it based only on pauses or are any other criteria?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by SandraRodgers** [comment, marked as answer]
Paragraphs are not determined by pauses in speech. There are a few things that determine a paragraph break: 

If there is a change in speaker or in channel, there will be a paragraph break. 

At a certain length (~200 words) / 5 sentences, there will be a paragraph break.

Hope this helps!
